web/modules/custom/joinup_community_content/js/pinned_content.js

```diff
@@ -12,7 +12,7 @@
    *   The HTML for the pinned content visual cue.
    */
   Drupal.theme.pinnedContentCue = function () {
-    return '<div class="listing__pin"><span class="icon icon--pin"></span></div>';
+    return '<div class="listing__corner listing__corner--pin"><span class="icon icon--pin"></span></div>';
```

Comment:
I like the idea of renaming the `listing__pin` to `listing__corner`.
Please remove `listing__corner--pin`, it's not necessary.

Reply:
On another, slightly related topic - when I both feature and pin a tile, only the pin shows and there is no featured tile behavior.

Reply:
Class has been removed.
Regarding this behaviour you need to contact Marco or Francesco, but I am quite sure that this is defined and implemented correctly. Tom replied to my text message that there is no possibility to have featured and pinned tiles at once.